### ISSUE TITLE
Docker build fails on COPY api/

### ISSUE THREAD

**Issue by bndw** [operator-framework/operator-sdk#4255]
## Bug Report

Following the [Golang Based Operator Tutorial](https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/), the step to [Build the Docker image](https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/#build-and-push-the-image) fails due to a bug in the Dockerfile.

#### What did you do?

`brew install operator-sdk` and followed the tutorial up to the point of building the Docker image:

```sh
make docker-build IMG=bndw/test-operator
```

#### What did you expect to see?

A Docker image successfully built.

#### What did you see instead? Under which circumstances?
A failed Docker step to `COPY` files:
```
Step 7/14 : COPY api/ api/
COPY failed: stat /var/lib/docker/tmp/docker-builder338415037/api: no such file or directory
make: *** [docker-build] Error 1
```

#### Environment

**Operator type:**

/language go 

**Kubernetes cluster type:**

Kind

`$ operator-sdk version`
```
operator-sdk version
operator-sdk version: "v1.2.0", commit: "215fc50b2d4acc7d92b36828f42d7d1ae212015c", kubernetes version: "v1.18.8", go version: "go1.15.4", GOOS: "darwin", GOARCH: "amd64"
```

#### Possible Solution

I was able to work around this by updating the Dockerfile to copy `./apis` instead of `./api`:

```diff
diff --git a/Dockerfile b/Dockerfile
index 74eb9d7..a765127 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -11,7 +11,7 @@ RUN go mod download
 
 # Copy the go source
 COPY main.go main.go
-COPY api/ api/
+COPY apis/ apis/
 COPY controllers/ controllers/
 ```

I'd be happy to submit a PR for the above patch if you can point me to the source Dockerfile (I couldn't find it in this repo tree).

**Comment by estroz** [comment]
@bndw if that changes works, then I assume your PROJECT file has `multigroup: true` as well? If so, did you run `operator-sdk edit --multigroup`? That command should have updated your Dockerfile appropriately.

**Comment by bndw** [comment]
@estroz Yes, I did add `multigroup: true` to the PROJECT file and didn't realize there was an additional command needed. 

I just reverted my manual change to the Dockerfile and tried your suggestion, but hit an error:
```
operator-sdk edit --multigroup
Error: unknown command "edit" for "operator-sdk"

Did you mean this?
	init

Run 'operator-sdk --help' for usage.
FATA[0000] unknown command "edit" for "operator-sdk"

Did you mean this?
	init
```

Either way, feel free to close this issue if you believe it's working as designed.

**Comment by estroz** [comment]
@bndw I referenced an unreleased command, whoops. `operator-sdk edit` was added in #4156 and will be released in v1.3.
